src/components/whiteframe/whiteframe.html

```diff
@@ -0,0 +1 @@
+<ng-content></ng-content>
```

Comment:
I believe Directive whould be better. But where should the css classes be loaded then?